wazo_webhookd/services/helpers.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2017 The Wazo Authors  (see the AUTHORS file)
```

Comment:
2019

Reply:
since this file was copied it would be 2017-2019. ok enough nitpicking for one day :door:

Reply:
done